packages/app/src/cli/models/extensions/specification.ts

```diff
@@ -13,6 +13,7 @@ export type ExtensionFeature =
   | 'cart_url'
   | 'esbuild'
   | 'single_js_entry_path'
+  | 'app_config'
```

Comment:
ah, again me not reading further down, I still feel as if these should be constants/enums but if its the only one included here then I digress

Reply:
✅ The `'app_config'` string has been extracted to a `APP_CONFIG_FEATURE` constant in 964290a7635d40561d94f2d381cbb1099d751833